### ISSUE TITLE
Member submissions: can anyone else object, and if so, can they object to acceptance?

### ISSUE THREAD

**Issue by dwsinger** [w3c/w3process#639]
In [Member Submission](https://www.w3.org/2021/Process-20211102/#SubmissionNo) the member who submitted can appeal a rejection.

But should it be possible for anyone else to appeal?  And if a submission is accepted that other members see as problematic, should it be possible to appeal acceptance?

(These are 'cleaning up loose ends' questions, I am not aware of current or past problems)

**Comment by michaelchampion** [comment]
Does the concept of a "member submission" still belong in the Process, considering the evolution of the patent policy, the near-universal use of "forkable" copyright licenses in web standards, and the use of community groups as the typical incubation mechanism?  It would seem cleaner to just remove member submissions to the process than try to untangle the Director from yet another corner case.

**Comment by dwsinger** [comment]
The [history](https://www.w3.org/Submission/) suggests that they remain an active tool.

**Comment by michaelchampion** [comment]
Well sure, if you expose a lever, people will yank on it.  Are they doing anything with that lever they couldn't do if it were removed?  Is repairing the mechanism the lever controls worth the trouble, considering you could just remove the lever?

**Comment by jeffjaffe** [comment]
As per the history that David cites, it has been useful particularly to get input from industry groups.

**Comment by michaelchampion** [comment]
Again, what input are you getting via the submission process you couldn't get from normal participation in WGs,  IGs or CGs?

I know from personal experience the Member Submission process was abused in the past to get the appearance of W3's imprimatur on "input from industry groups" that had no broad industry support or prospects of getting into a Recommendation. I don't know of that happening in the last 10 years or so, but I don't see many submissions that had a real impact since the FIDO submission in 2015 either. 

This is a pretty minor nit in the Process so I don't really care what happens to this issue, but I am concerned that the Process is accreting ever more complexity to cover anachronistic corner cases.  Assuming W3C survives the LE transition and the AB can focus on process and community issues next year, cleaning up this sort of legacy cruft might be worth some attention.

**Comment by mnot** [comment]
I tend to agree with Michael here - member submissions made sense pre-CG, much less so now.

**Comment by fantasai** [comment]
If you want to suggestion removing Member submissions, _please file a new issue_. That's not this issue.

**Comment by michaelchampion** [comment]
I proposed resolving the issue of how to appeal member submission decisions by removing member submissions🤷

**Comment by palemieux** [comment]
@michaelchampion Say a group of folks that a not W3C members come up with a spec, and would like to continue the work in W3C, how would they get started?

**Comment by michaelchampion** [comment]
Create a Community Group; have the WG publish the spec with an open copyright license and patent commitments from the contributors; build a community of users who have problems the spec solves, implementers who will support the standard in their products, tools, frameworks, etc.; then draft a charter for a new WG or discuss expanding the charter of an existing WG.

**Comment by palemieux** [comment]
Is there any way for the contribution, with the relevant patent commitments, to be directly accepted by a WG?

**Comment by michaelchampion** [comment]
I don't see why not, but I haven't thought about this for a while.  If someone writes a spec, joins a WG for which it is in scope thus making patent commitments to anything the WG takes to Rec, proposes it and the WG accepts it, I don't foresee any problems.  

Member submissions historically were for situations where there wasn't an existing WG with the spec in scope, and before there were CGs with a patent policy designed to cover such incubation scenarios.

To be clear, I don't have strong feelings on whether member submissions should be removed from the process.  I'm just suggesting the Process CG convince themselves that member submissions are worth fixing to work in a director-free world as opposed to just removing them from the Process so this issue doesn't arise.

**Comment by frivoal** [comment]
As requested by the chair, I moved all the comments about removing member submissions from the Process into a new issue: #648. Please keep this discussion on topic.

**Comment by fantasai** [comment]
I think the Process is clear that only the Submitter(s) can initial an appeal. Whether we want to allow ACs to appeal acknowledgement of a Member Submission is an interesting question, but do we really need it?

**Comment by brentzundel** [comment]
The Revising W3C Process CG just discussed `Subtopic: Issue 639`, and agreed to the following:

* `RESOLVED: Remove the "propose to close" issue`

<details><summary>The full IRC log of that discussion</summary>
&lt;brent> Subtopic: Issue 639<br>
&lt;brent> Github: https://github.com/w3c/process/issues/639<br>
&lt;Ian> q+<br>
&lt;brent> ack Ian<br>
&lt;plh> q+<br>
&lt;Ian> Ian: I think the AB should review this whole mechanism before addressing an issue that is an abstract question.<br>
&lt;Ian> Brent: That's a good question, but the issue is saying "Since they do exist...let's examine this topic"<br>
&lt;plh> ack plh<br>
&lt;Ian> PLH: We ended up simplifying the submission process.<br>
&lt;Ian> PLH: It's hard to remember the rationale for the appeal<br>
&lt;Ian> ...I'd be in favor of allowing others from the AC to appeal<br>
&lt;Ian> Brent: I am hearing that we should not close this issue.<br>
&lt;Ian> RESOLVED: Remove the "propose to close" issue<br>
</details>
